tools/spades.py

```diff
@@ -29,7 +29,7 @@ class SpadesTool(tools.Tool):
     def __init__(self, install_methods=None):
         if install_methods is None:
             install_methods = [tools.CondaPackage(TOOL_NAME, version=TOOL_VERSION, executable='spades.py',
-                                                  verifycmd='echo spades.py --help', env='spades_env')]
+                                                  verifycmd='spades.py --version')]
```

Comment:
Even though we've had this ability to separate conda environments, in truth, I don't think it really works that well. Putting spades in a non-default conda env means that Travis installs it twice (once in the default, and again in the special env), and building a conda package for viral-ngs requires that the conda dependencies have to be compatible anyway...

And since it seems like the spades conda package is coexisting just fine with everything else, both on my Mac and on Travis's environments, it's probably fine at this point.